Actions/Github-Helper.psm1

```diff
@@ -153,7 +153,7 @@ function GetDependencies {
                     if (Test-Path $downloadName -PathType Container) {
                         $folder = Get-Item $downloadName
                         Get-ChildItem -Path $folder | ForEach-Object {
-                            if ($mask -eq 'TestApps') {
+                            if ($mask -like '*TestApps') {
```

Comment:
Shouldn't this be -eq "$($buildMode)TestApps" ?

Reply:
Only if buildMode is not default I guess.

Reply:
There's already `notlike '*TestApps'` so I think it's fine.